package.json

```diff
@@ -292,14 +290,21 @@
 	},
 	"resolutions": {
 		"react-dates": "17.1.1",
+		"@types/wordpress__block-editor": "npm:noop-package@1.0.0",
+		"@types/wordpress__editor": "npm:noop-package@1.0.0",
+		"@types/wordpress__notices": "npm:noop-package@1.0.0",
+		"@types/wordpress__rich-text": "npm:noop-package@1.0.0",
 		"@types/react": "^17.0.39",
 		"newspack-components/@wordpress/base-styles": "4.5.0",
 		"newspack-components/@wordpress/components": "19.15.0",
 		"newspack-components/@wordpress/element": "4.7.0",
 		"newspack-components/@wordpress/i18n": "4.9.0",
 		"keytar@npm:7.7.0/node-addon-api": "3.1.0",
 		"lzma-native": "^8.0.5",
-		"@wordpress/interface@^4.8.0": "patch:@wordpress/interface@npm%3A4.8.0#./.yarn/patches/@wordpress-interface-npm-4.8.0-8a39ad37cf.patch"
+		"@wordpress/base-styles": "4.13.0",
+		"@wordpress/interface@^4.8.0": "patch:@wordpress/interface@npm%3A4.8.0#./.yarn/patches/@wordpress-interface-npm-4.8.0-8a39ad37cf.patch",
+		"@wordpress/data-controls": "2.22.0",
+		"@wordpress/element": "4.20.0"
```

Comment:
This package has been a bit merciless with breaking changes sneaking into minor versions, so we need to pin it down to avoid them inadvertently sneaking into the code.